llvm/tools/llvm-objcopy/CopyConfig.cpp

```diff
@@ -269,6 +269,9 @@ static const StringMap<MachineInfo> TargetMap{
     {"elf32-tradbigmips", {ELF::EM_MIPS, false, false}},
     {"elf32-tradlittlemips", {ELF::EM_MIPS, false, true}},
     {"elf64-tradbigmips", {ELF::EM_MIPS, true, false}},
+    {"elf64-tradbigmips-fbsd-cheri",
```

Comment:
objcopy already sets `ELF::ELFOSABI_FREEBSD` if the name ends in "-freebsd" (https://github.com/CTSRD-CHERI/llvm-project/blob/fb6893937dad46a2e55646677c0bfa157caa1c7b/llvm/tools/llvm-objcopy/CopyConfig.cpp#L285)

Would "elf64-tradbigmips-cheri-freebsd" also work?

Reply:
Or maybe -purecap-freebsd, since there is also an e_flag for "supports CHERI"

Reply:
Definitely better